tools/sdk-versioner/src/main.rs

```diff
@@ -50,7 +50,7 @@ impl Opt {
 }
 
 fn main() -> anyhow::Result<()> {
-    let opt = Opt::from_args().validate()?;
+    let opt = Args::parse().validate()?;
```

Comment:
It looks like CI is failing since the Docker image tests that all the tools are in the right place by calling them with their `--version` flag, and I guess `structopt` must have been giving us that flag for free.

Reply:
Thanks! Fixed in https://github.com/awslabs/smithy-rs/pull/1293/commits/5365a83ce62412447d0f029ae40ebafff908b855